src/pages/MarkingPage/Scrollspy/ScrollspyItem/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Badge, Flex, Link, Separator } from '@radix-ui/themes'
+import classNames from 'classnames'
+import { isNil } from 'lodash'
+import React, { FC } from 'react'
+
+import { NO_MARK } from '../../constants'
+import './index.css'
+
+export interface ScrollspyItemProps {
+  id: string
+  active: boolean
+  label: string
+  partial?: number
```

Comment:
Rename to `partMarks` and `totalMarks` for more clarity?

Reply:
This is a good suggestion but the ScrollspyItem is used for both Question and Section. For question, this is partial _marks_ over total _marks_, for a section this is _current mark_ over _maximum mark_. To avoid inconsistency, I have generalised by a single step and made this into a more generic indicator of partial/total. Don't have a strooong opinion on it though.